spec/classes/init_spec.rb

```diff
@@ -1,11 +1,15 @@
 require 'spec_helper'
 
 describe 'kibana', :type => :class do
-  let(:facts) { {
+
+  let :facts do {
+    :operatingsystem => 'CentOS',
```

Comment:
I think this is the only place "do { ... } end" is used instead of "{{ ... }}".  Nitpicky, but just trying to keep it consistent - must have missed it before, sorry.